ibis/mapd/tests/test_operations.py

```diff
@@ -0,0 +1,35 @@
+import ibis
+import numpy as np
+import pytest
+
+from pytest import param
+
+pytestmark = pytest.mark.mapd
+pytest.importorskip('pymapd')
+
+
+@pytest.mark.parametrize(('result_fn', 'expected'), [
+    param(
+        lambda t: t[t, ibis.literal(1).degrees().name('n')].limit(1)['n'],
+        57.2957795130823,
+        id='literal_degree'
+    ),
+    param(
+        lambda t: t[t, ibis.literal(1).radians().name('n')].limit(1)['n'],
+        0.0174532925199433,
+        id='literal_radians'
+    ),
+    param(
+        lambda t: t.double_col.corr(t.float_col),
+        1.000000000000113,
+        id='double_float_correlation'
+    ),
+    param(
+        lambda t: t.double_col.cov(t.float_col),
+        91.67005567565313,
```

Comment:
Ultimately we will want to change these to use a pandas call or numpy call, so that we don't have to depend on hard coded values.

This is fine for now though.